libs/i18n-labels/source/datasets.json

```diff
@@ -1034,6 +1034,8 @@
   "public-eez-land": {
     "name": "EEZ (marineregions.org)",
     "description": "Flanders Marine Institute (2019). Maritime Boundaries Geodatabase: Maritime Boundaries and Exclusive Economic Zones (200NM), version 11. Source: marineregions.org",
+    "dataWarning": "This platform uses EEZ reference layers from Marine Regions",
```

Comment:
this file is going to be overwritten by the [script that autogenerates](https://github.com/GlobalFishingWatch/crowdin-datasets-internationalization) this file based on the api datasets status (basically: config.translate = true).
Now it runs manually and we could keep it by hand but it is going to be automate soon so we need to:
- include these properties on the dataset configuration
or
- move these to the source translations app scope and ensure the id matches using [this config](https://github.com/GlobalFishingWatch/frontend/pull/1268/files#diff-6ac73904c9a979f3e0150f088b8639608618633f9a93689539e065bdfd3bfb27R26)